src/test/java/no/unit/nva/model/FileTest.java

```diff
@@ -45,10 +46,21 @@ void fileWithAdministrativeAgreementAndNoLicenseIsValid() {
     @Test
     void fileValidationWithNoLicenseAndNotAdministrativeAgreementThrowsMissingLicenseException() {
         File file = new File(UUID.randomUUID(), FILE_NAME, MIME_TYPE, FILE_SIZE, null, false, true, null);
-        MissingLicenseException exception = assertThrows(MissingLicenseException.class, () -> {
-            file.validate();
-        });
-
+        MissingLicenseException exception = assertThrows(MissingLicenseException.class, file::validate);
         assertEquals(File.MISSING_LICENSE, exception.getMessage());
     }
+
+    @DisplayName("A file that is not an administrative agreement and has a license validates")
```

Comment:
A file validates when it is not an administrative agreement and it has a license.